.travis/centos_7.sh

```diff
@@ -10,7 +10,7 @@ ${DOCKER_EXEC} ./autogen.sh
 ${DOCKER_EXEC} ./configure
 ${DOCKER_EXEC} make dist
 ${DOCKER_EXEC} /bin/sh -c 'cp -fv pbspro-*.tar.gz /root/rpmbuild/SOURCES/'
-${DOCKER_EXEC} rpmbuild -bb pbspro.spec
+${DOCKER_EXEC} /bin/sh -c 'CFLAGS="-g -O2 -Wall -Werror" rpmbuild -bb pbspro.spec'
```

Comment:
The "-g -O2" were used by default when no CFLAGS were specified. We want to reuse them and add "-Wall -Werror" to achieve the same build as before, but with more warnings enabled. To answer your question... we want to specify -g because it's always been there.